zerver/data_import/import_util.py

```diff
@@ -825,3 +827,21 @@ def long_term_idle_helper(
             user_profile_row["last_active_message_id"] = 1
 
     return long_term_idle
+
+
+def check_converted_users_email(user_emails: list[str]) -> None:
```

Comment:
This function is here and not in `user_handler.py` inside `check_user_emails()` because unlike Mattermost and Rocket.chat importers, the Slack importer doesn't use the `UserHandler` class.

Reply:
Maybe `check_imported_user_emails`? Or `validate_imported_user_emails`?

Reply:
Or `validate_user_emails_for_import`?

Reply:
hmm, I guess `validate_user_emails_for_import` is ok. I avoided using the word "imported" because at this point we're still converting the Slack export.